#! /bin/sh /usr/share/dpatch/dpatch-run
## nomarch.dpatch by Kilian Krause <kilian@debian.org>
##
## DP: Use -O2 instead of -O6

@DPATCH@
--- a/formats/mp3/Makefile
+++ b/formats/mp3/Makefile
@@ -8,12 +8,6 @@
 
 ifeq (${OSARCH},Linux)
 PROC=$(shell uname -m)
-ifeq ($(PROC),x86_64)
-# You must have GCC 3.4 to use k8, otherwise use athlon
-PROC=k8
-#PROC=athlon
-#OPTIONS+=-m64
-endif
 ifeq ($(PROC),sparc64)
 #The problem with sparc is the best stuff is in newer versions of gcc (post 3.0) only.
 #This works for even old (2.96) versions of gcc and provides a small boost either way.
@@ -55,7 +49,7 @@
 endif
 
 #Tell gcc to optimize the asterisk's code
-OPTIMIZE+=-O6
+OPTIMIZE+=-O2
 
 CFLAGS+=$(OPTIMIZE)
 CFLAGS+= $(OPTIONS)
